gocd/sp.target.gocd.yaml

```diff
@@ -167,3 +167,42 @@ pipelines:
           - staging-bot
         tasks:
           - script: osc -A https://api.suse.de release SUSE:SLE-15-SP5:GA:TEST
+
+  SLE15.SP5.ReleaseScc:
+    parameters:
+      product: sle15sp5
+      target: MyTarget
+      build: "0.0"
+    stages:
+    - Release.Images.Update:
+        approval: manual
+        roles:
+          - SLE
+        resources:
+          - staging-bot
+        tasks:
+          - script: curl --verbose --max-time 3600 http://10.160.0.10:5000/main/update?product=#{product}&target=#{target}&build=#{build}
+    - Release.Images.Rename:
+        approval: manual
+        roles:
+          - SLE
+        resources:
+          - staging-bot
+        tasks:
+          - script: curl --verbose --max-time 3600 http://10.160.0.10:5000/main/rename?product=#{product}&target=#{target}
+    - Release.Images.Update:
+        approval: manual
+        roles:
+          - SLE
+        resources:
+          - staging-bot
+        tasks:
+          - script: curl --verbose --max-time 3600 http://10.160.0.10:5000/main/sync?product=#{product}&target=#{target}
+    - Release.Images.Update:
+        approval: manual
```

Comment:
that are a lot of manual steps. You're certain you want the user to click 5 times?

Reply:
Yes as this is currently in beta status and thus I want this to be manual until it was tested. The curl call is limited to the botmaster host and thus I can't test it locally from my machine.

Reply:
How I love things that can only be tested in production :)